com.mastermindzh.tidal-hifi.yml

```diff
@@ -0,0 +1,43 @@
+app-id: com.mastermindzh.tidal-hifi
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+sdk: org.freedesktop.Sdk
+separate-locales: false
+command: com.mastermindzh.tidal-hifi
+rename-desktop-file: tidal-hifi.desktop
+finish-args:
+  - '--share=ipc'
+  - '--socket=x11'
+  - '--socket=pulseaudio'
+  - '--share=network'
+  - '--device=dri'
+  - '--talk-name=org.freedesktop.Notifications'
+  - '--filesystem=xdg-run/app/com.discordapp.Discord:create'
+  - '--own-name=org.mpris.MediaPlayer2.tidal-hifi'
+  - '--talk-name=org.freedesktop.Notifications'
+modules:
+  - name: tidal-hifi
+    buildsystem: simple
+    build-commands:
+      - chmod +x tidal-hifi-2.8.0.AppImage
+      - ./tidal-hifi-2.8.0.AppImage --appimage-extract
+
+      - install -dm755 "${FLATPAK_DEST}/bin" "${FLATPAK_DEST}/lib/tidal-hifi"
+      - cp -r squashfs-root/* "${FLATPAK_DEST}/lib/tidal-hifi"
+      - cp -r squashfs-root/usr/lib/* "${FLATPAK_DEST}/lib"
+      
+      - install -Dm644 "com.mastermindzh.tidal-hifi.png" "${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/com.mastermindzh.tidal-hifi.png"
```

Comment:
the PNG is 512x512. Since you include it, better make it the right size instead of misreprensenting its size.